docs/posts/flink-quickstart/index.qmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Ibis goes real-time! Introducing the new Flink backend for Ibis"
+author: "Deepyaman Datta"
+date: "2024-01-22"
+categories:
+    - blog
+    - flink
+    - stream processing
+---
+
+## Introduction
+
+Ibis 8.0.0 marks the official release of the Apache Flink backend for Ibis. Flink is one of the most established stream-processing frameworks out there and a central part of the real-time data infrastructure at companies like DoorDash, LinkedIn, Netflix, and Uber. The Flink backend is also the first streaming backend Ibis supports. Follow along as we define and execute a simple streaming job using Ibis!
+
+## Installation prerequisites
+
+* **Docker Compose:** This tutorial uses Docker Compose to manage an Apache Kafka environment (including sample data generation) and a Flink cluster (for [remote execution](#remote-execution)). You can [download and install Docker Compose from the official website](https://docs.docker.com/compose/install/).
+* **JDK 11:** Flink requires Java 11. If you don't already have JDK 11 installed, you can [get the appropriate Eclipse Temurin release](https://adoptium.net/temurin/releases/?package=jdk&version=11).
+* **Python:** To follow along, you need Python 3.9 or 3.10.
+
+## Installing the Flink backend for Ibis
+
+We use a Python client to explore data in Kafka topics. You can install it, alongside the Flink backend for Ibis, with `pip`, `conda`, `mamba`, or `pixi`:
+
+::: {.panel-tabset}
```

Comment:
if only `pip` is supported right now, can we get rid of this tabset? it's a bit odd to have 3 others that aren't supported

Reply:
They should all be supported by 8.0 release; it will just require adding it to the Ibis feedstock (which @gforsyth will do after releasing, I believe). I can mark the other 3 not supported until then here, too, if that would be better?